earth_enterprise/src/fusion/autoingest/StorageManager_unittest.cpp

```diff
@@ -580,6 +580,36 @@ TEST_F(StorageManagerTest, InvalidRef) {
   ASSERT_FALSE(asset) << "Invalid refs should return empty handles";
 }
 
+TEST_F(StorageManagerTest, PruneLimiting) {
+  storageManager.SetPrunePercent(.5);
+
+  PointerType newItem = make_shared<TestItem>();
+  ASSERT_EQ(storageManager.CacheSize(), 0) << "Storage manager has unexpected item in cache";
+  ASSERT_EQ(storageManager.DirtySize(), 0) << "Storage manager has unexpected item in dirty map";
+
+  storageManager.AddNew("newItem", newItem);
+  ASSERT_EQ(storageManager.CacheSize(), 1) << "Storage manager has wrong number of items in cache";
+  ASSERT_EQ(storageManager.DirtySize(), 1) << "Storage manager has wrong number of items in dirty map";
+  ASSERT_EQ(storageManager.DetermineIfPrune(), false) << "Storage manager has determined wrong choice for pruning cache";
```

Comment:
Minor: there is an `ASSERT_FALSE` (and an `ASSERT_TRUE` for the other `DetermineIfPrune` check).